.pre-commit-config.yaml

```diff
@@ -37,3 +37,11 @@ repos:
     - id: requirements-txt-fixer
     - id: sort-simple-yaml
     - id: trailing-whitespace
+# - repo: https://github.com/PyCQA/pylint
+#   rev: pylint-1.9.4
+#   hooks:
+#     - id: pylint
+# - repo: https://github.com/asottile/dead
+#   rev: v1.3.0
+#   hooks:
+#     - id: dead
```

Comment:
Do you mean to be adding commented-out lines to this file?

Reply:
Thanks for that!

Reply:
Yes. The problem is that these tools (at least via easy pre-commit hook setups) require python3. I was investigating them before coming to this conclusion, so left them commented out as a reminder to re-enable them once the python3 migration is done.

In particular, pylint didn't support pre-commit out-of-the-box until 2.x but they dropped python 2 support in 2.x, so the hook fails to install for older versions. Dead seems to have only ever supported Python3 (it's a static analysis tool to help find dead code).

There are hooks for mypy and pyupgrade that I'll be adding as I merge the python3 and type-checking branches into unstable. I've had a parallel development track, but am now bringing this all together.